webapp/lib/controller.dart

```diff
@@ -204,21 +212,37 @@ bool isProjectStale(model.NeedsReplyData projectData) {
   }
 }
 
-void setupProjectTimer(model.NeedsReplyData projectData, [bool stale = false]) {
-  projectTimers[projectData.project]?.cancel();
+String getProjectTimer(Object data) {
```

Comment:
as above - the timer is no longer project-only, so perhaps we should rename all variables related to it to what the time is doing? it's basically a watchdog timer, so we could have:
`getProjectTimer` -> `getWatchdogTimer`,
`setupProjectTimer` -> `setupWatchdogTimer`,
`projectTimers` -> `watchdogTimers` etc.